pico/testing/src/main/java/io/helidon/pico/testing/PicoTestingSupport.java

```diff
@@ -140,11 +135,6 @@ private static class Internal extends PicoServicesHolder {
         public static void reset() {
             PicoServicesHolder.reset();
             instance = lazyCreate(basicTestableConfig());
-
```

Comment:
Because there is now a common support for resettable components that can be registered with `Pico`, so when you call reset on pico services holder, all of these components are reset as part of that process.